packages/react-components/react-positioning/src/utils/normalizeAutoSize.ts

```diff
@@ -0,0 +1,34 @@
+import type { NormalizedAutoSize, PositioningOptions } from '../types';
+
+/**
+ * AutoSizes contains many options from historic implementation.
+ * Now options 'always'/'height-always'/'width-always' are obsolete.
+ * This function maps them to true/'height'/'width'
+ */
+export const normalizeAutoSize = (autoSize?: PositioningOptions['autoSize']): NormalizedAutoSize | false => {
+  switch (autoSize) {
+    case 'always':
```

Comment:
let's quickly add some tests here... we don't want the risk of breaking option normalization

Reply:
I was thinking about testing normalizeAutoSize but I got so lazy 🙈, so I bet myself, if you leave a comment asking me to test it, I will have some extra chocolate today. And now I'm writing test and eating chocolate :D